sandbox/yitan-jiahao-grafana-middleware-2019/SampleRatio.py

```diff
@@ -0,0 +1,161 @@
+import requests
+import json
+import numpy
+import matplotlib.pyplot as plt
+import math
+
+from tslearn.piecewise import SymbolicAggregateApproximation
+from tslearn.utils import to_time_series
+
+from tslearn.preprocessing import TimeSeriesScalerMeanVariance
+from tslearn.preprocessing import TimeSeriesResampler
+
+
+##dbname = "NOAA_water_database"
```

Comment:
If the commended code is unnecessary, please delete them

Reply:
This file is highly dependent on the dataset. We used it on 3 different datasets. The urls need to be modified every time we change a db, so it's more convenient to leave it there.